srcpkgs/urh/template

```diff
@@ -13,7 +13,7 @@ maintainer="Emil Miler <em@0x45.cz>"
 license="GPL-3.0-or-later"
 homepage="https://github.com/jopohl/urh"
 distfiles="${homepage}/archive/v${version}.tar.gz"
-checksum=a36dd3ff79f66cc0a746a0b57aaf278c86dda7c091a3b10682fe4fcaa7e91586
+checksum=d2d31df8819489910e95451dcb8faf3aa5c371da65bd9df3a4712e532186b67f
 make_check=no # tests/test_advanced_modulation_settings.py Aborted (core dumped)
```

Comment:
if this is still happening, can we just disable the one test that's causing the issues or just remove this file and let the rest run?

Reply:
The testsuite fails, but it seems like some libraries for `checkdepends` are missing? I am not sure it's a best practice to add `pip` as a dependency for check, but in the output seems to complain about that.

Reply:
Adding pip to checkdepends only leads to it installing all dependencies it needs in the versions it needs with pip. We don't want that, but it's also not telling us pip is missing:
```
error: Command '['/usr/bin/python3', '-m', 'pip', '--disable-pip-version-check', 'wheel', '--no-deps', '-w', '/tmp/tmp6dbhqicv', '--quiet', 'psutil']' returned non-zero exit status 1.
```
If you read this carefully, it's telling us that psutil is missing. Most of these missing dependencies can be probably solved with a simple `checkdepends="$depends"`

Reply:
Alright, makes sense. I originally thought the dependencies get installed from `makedepends` and tests are being run after that.

Reply:
One of the tests is taking way too long and CI times out. It would be nice if we could figure out which one it is and disable it specifically, but disabling all tests and modifying the comment next to make_check=no to match reality would also be acceptable.

Reply:
Sorry for taking too long, I've had a lot of work lately. I'd rather have it updated quickly, so i disabled the tests entirely for now. I will try and debug it in the future.